### ISSUE TITLE
docker v.3.0 blank sites and missing features

### ISSUE THREAD

**Issue by frapit** [Kitware/CDash#1106]
I just installed the docker version of cdash on my server. It runs via `docker-compose` behind a simple traefik reverse proxy configuration which also adds TLS support. I notice that some of the pages are not working. I get a blank screen when trying to create a new project as well as when I click on a test to see its output. Maybe this is related to #1098. When I force http for this specific page it works. Also I am missing that little button next to a build, which allows one to remove or tag that build. The version of cdash (v3.0.2-11-g70eb0da9d) is the same as on [my.cdash.org](my.cdash.org) so I am wondering why they differ.

Any clues how to fix this and make it work with https? 

Thanks!

**Comment by zackgalbreath** [comment]
Thanks for the report.

Can you check if you have a value for `APP_URL` set in your .env file?

**Comment by RapIt** [comment]
Thanks for the reply. Yes, the `APP_URL` variable is set to my url and the `MIX_APP_URL` is also set. 
I tried with both the registry container and  a manually built container. I also tried to build it with `$CDASH_USE_HTTPS = '1';
` in `app/cdash/config/config.php` but it did not seem to make any difference.

**Comment by martin5233** [comment]
I encountered the same problem in conjunction with a Traefik proxy. For me even the initial redirect from the root URL of the installation to viewProjects.php does not work, as it does not seem to consider the configured CDASH_BASE_URL. I have also checked the APP_URL, which is correct. In my case I have configured Traefik to forward requests under https://host/CDash to the docker container with the CDash URL part removed, but the redirection needs to add the CDash part to the redirected URL, which does not occur.
